src/Fable.Transforms/Replacements.Util.fs

```diff
@@ -254,6 +256,41 @@ let makeRuntimeType genArgs fullName =
 let makeFSharpCoreType genArgs fullName =
     makeDeclaredType "FSharp.Core" genArgs fullName
 
+let makeStringTemplate tag (str: string) (holes: {| Index: int; Length: int |}[]) values =
+    let mutable prevIndex = 0
+    let parts = [
+        for i = 0 to holes.Length - 1 do
+            let m = holes.[i]
+            let strPart = str.Substring(prevIndex, m.Index - prevIndex)
+            prevIndex <- m.Index + m.Length
+            strPart
+        str.Substring(prevIndex)
+    ]
+    StringTemplate(tag, parts, values)
+
+let makeStringTemplateFrom values = function
+    | StringConst str ->
+        // In the case of interpolated strings, the F# compiler doesn't resolve escaped %
+        // (though it does resolve double braces {{ }})
+        let str = str.Replace("%%" , "%")
+        (Some [], Regex.Matches(str, @"((?<!%)%(?:[0+\- ]*)(?:\d+)?(?:\.\d+)?\w)?%P\(\)") |> Seq.cast<Match>)
+        ||> Seq.fold (fun acc m ->
+            match acc with
+            | None -> None
+            | Some acc ->
+                // TODO: If arguments need format, format them individually
+                let doesNotNeedFormat =
+                    not m.Groups.[1].Success
+                    || m.Groups.[1].Value = "%s"
```

Comment:
@alfonsogarciacaro Can we parameterize these so they can be passed in as an array, e.g. see [here](https://github.com/fable-compiler/Fable/blob/beyond/src/Fable.Transforms/Rust/Replacements.fs#L955).